apps/files/src/components/FilesLists/StatusIndicators/DefaultIndicators.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div>
+    <oc-button
+      v-for="(indicator, index) in indicators"
+      :key="index"
+      class="file-row-share-indicator uk-text-middle"
+      :class="{ 'uk-margin-xsmall-left' : index > 0 }"
```

Comment:
did you change / adjust styling here ?

I didn't know that it's possible to have an attribute twice this way

Reply:
Added just the margin class.

Should work as long as one of them is `v-bind`. Not sure though if this is only for the class. Haven't yet tried it with anything else.